### ISSUE TITLE
Air Conditioning Fans

### ISSUE THREAD

**Issue by fadandr** [evgolsh/homebridge-crestron-home#6]
This plugin for Crestron is the best. Super simple and maximum useful. But if it would have support not only of Lights, but as minimum AC too, it will become unbelievable. 

Crestron setuppers in my country quite lazy and expensive, I tried to ask them to connect Crestron to my Apple Home, but nothing happened. So if the author will update this plugin to support Climate Control in Crestron, my life will become much easier. Moreover, if this work should be paid, I am completely ready for it. So, please, Eugene, could you add Climate Control to your plugin?

**Comment by evgolsh** [comment]
Hey, what is AC system are you using? I never developed it because there are plenty of plugins for the Homebridge that can be controlled in Apple Homekit directly. You are the first who is asking for this.

**Comment by fadandr** [comment]
The problem is, that my Daikin ACs if currently inside Crestron system. I haven't possibility to manipulate it without Crestron. So currently I setup through your plugin and scenes. But I would be completely happy if we can manipulate crestron ACs through your plugin as it is with lights now. Scenes are ok, but I like more customized icons -))))

**Comment by evgolsh** [comment]
Daikin should be plugged into Crestron via something like CoolMasterNet. If yes, there are Homebridge plugins.
I'll have a look what does it take to implement it in my plugin.

**Comment by fadandr** [comment]
Yes, you right. It is interconnected with CoolMasterNet, and we have even a plug-in for it. And I spent a lot of time trying to make it work, but completely unsuccessful. And I tried all other plugins for Crestron, Daikin and others. No one is so simple and useful as yours, and no one is working with ACs. Some of them expose homekit devices to crestron. But my task is different, to expose crestron devices to homekit 🙂

**Comment by wolfiegrr** [comment]
I think exposing Thermostats would be AMAZING.  Our thermostats are connected through Crestron directly, and are only exposed through the system.  I have currently no way of getting them on Homekit or Homebridge.

**Comment by gordonxman** [comment]
I have a Crestron Home system and use the Creston Home plugin daily. We also have Daikin HVAC with CoolMasterNet and successfully use this additional plugin to connect to HomeKit. 

https://github.com/nitaybz/homebridge-cool-automation

Both work well together and provide integrated control in the Home App.

**Comment by wolfiegrr** [comment]
My thermostat I believe is the Crestron branded CHV-TSTATW
https://www.crestron.com/Products/Lighting-Environment/Climate-Control/Thermostats/CHV-TSTATW
I don't believe this one can be seperately controlled outside of Crestron (as far as my understanding goes).

**Comment by wolfiegrr** [comment]
Following up on my last comment, and similarly with fadandr, I am interested in paying for someone's time to make this enhancement.  It's one of the key automations in Crestron I cannot manage from Homekit.

**Comment by smart** [comment]
Needed this as well and made a fork with thermostat and door lock support, will submit as PR, local testing worked but still getting setup to seamlessly cut over and test with homekit

**Comment by evgolsh** [comment]
Merged and published 1.1.10 with thermostats support and other major changes by @smart 
@wolfiegrr see if it works for you, it does discover my ACs now, thanks to @smart

**Comment by wolfiegrr** [comment]
@evgolsh - Works like a Charm!  Thank's @smart -- this has been a huge boost to functionality for me, I truly appreciate it.
